src/js/app.js

```diff
@@ -149,14 +180,20 @@ app.config(['$stateProvider', '$urlRouterProvider', '$locationProvider', '$httpP
         templateUrl: 'partials/news/news-create.html',
         controller: 'NewsCreateCtrl',
         requireLogin: true,
-        permission: 'news'
+        permission: 'news',
+        meta: {
```

Comment:
You still want to nested the `meta` object map underneath a `data` object map in order for data inheritance via `ui-router` and `ng-meta` to work:
https://github.com/vinaygopinath/ngMeta#data-inheritance-in-ui-router

Also a missing colon here, that's the source of the failing Travis CI build.